lua/lsp/lsp.lua

```diff
@@ -0,0 +1,40 @@
+local M = {}
+local utils = require('lsp/utils')
+
+function M.get_allowed_servers(buffer_filetype, servers)
```

Comment:
we should change this to `get_allowed_servers.lua` but feel free to directly rexport it in the main lua module.

```lua
M.get_allowed_servers = require './get_allowed_servers'
```